pkgs/development/python-modules/isoweek/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "isoweek";
+  version = "1.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1s7zsf0pab0l9gn6456qadnz5i5h90hafcjwnhx5mq23qjxggwvk";
+  };
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "The module provide the class Week. Instances represent specific weeks spanning Monday to Sunday.";
+    homepage = https://github.com/gisle/isoweek;
```

```suggestion
    homepage = "https://github.com/gisle/isoweek";
```
Hint: We're probably going to deprecate unquotes URL syntax when RFC 45 is approved.

---

updated everything per your suggestions, thanks!